exercises/nucleotide-count/description.md

```diff
@@ -1,25 +1,13 @@
-Given a DNA string, compute how many times each nucleotide occurs in the string.
+Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
```

Comment:
This sentence looks like it covers the point you raised today (9/28/17) in #902.  The rest of the changes are interesting, and I don't think they are bad, but I am not sure that they are necessary.  I am curious to read other peoples thoughts on this.

Reply:
I made an effort not to bring in much new information. I simply structured the sentences and information already there to be more precise and accurate in accordance with biochemical discourse.
You may use my changes as you like :)